src/backend/distributed/master/master_repair_shards.c

```diff
@@ -109,6 +126,51 @@ master_move_shard_placement(PG_FUNCTION_ARGS)
 
 
 /*
+ * LookupShardReplicationMode maps the oids of citus.shard_replication_mode enum
+ * values to a char.
+ */
+static char
+LookupShardReplicationMode(Oid shardReplicationModeOid)
+{
+	HeapTuple enumTuple = NULL;
+	Form_pg_enum enumForm = NULL;
+	char shardReplicationMode = 0;
+	const char *enumLabel = NULL;
+
+	enumTuple = SearchSysCache1(ENUMOID, ObjectIdGetDatum(shardReplicationModeOid));
```

Comment:
Minor: Might simplify the code if we use `enum_out` with the direct function call?